src/style.css

```
@@ -1,3 +1,64 @@
-body {
+/* stylelint-disable color-hex-case */
+/* stylelint-disable string-quotes */
+html {
+  font-family: 'Roboto', Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+.body {
+  min-width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
   margin: 0;
 }
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  cursor: pointer;
+  padding: 23px 0; /* як це прописати замість тупого підбору? */
+  /* stylelint-disable-next-line color-hex-length */
+  color: #000000;
+}
+
+.nav__list > :not(:last-child) {
+  margin-right: 20px;
+}
+
+.is-active {
+  color: #00ACDC;
+}
```

```suggestion
.is-active,
.nav__link:hover {
  color: #00ACDC;
}
```
you may list selectors with same properties - FYI